### PR TITLE
DIAC-566 - 3.2.3: Consistent Navigation (Level AA)

### PR DESCRIPTION
**Pull request by pr00279**
### Jira link

https://tools.hmcts.net/jira/browse/DIAC-566

### Change description

Ensure that "Skip to Main Content" link brings users to a consistent landmark across all pages

### Testing done

Attached to DIAC-566 in video format

### CODE REVIEW

**Review by jamiephillipshmcts — APPROVED** [review]

**Review by jamiephillipshmcts — COMMENTED** [review]
Might be worth extracting this common functionality into a function so not all repeated: const env = nunjucks.configure([
    'node_modules/govuk-frontend',
    'views'
  ]);

  env.addFilter('eval', function(text: string) {
    return nunjucks.renderString(text, this.ctx);
  });

  env.addGlobal('i18n', i18n);

File: test/accessibility/skip-link.test.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+  env.addFilter('eval', function(text: string) {
+    return nunjucks.renderString(text, this.ctx);
+  });
+
+  env.addGlobal('i18n', i18n);
+
+  const skipToMainContentLink = '<a href="#main-content" class="govuk-skip-link" data-module="govuk-skip-link">Skip to main content</a>';
+
+  it('should render the "Skip to main content" link in layout.njk with href="#main-content"', () => {
```
Comment (jamiephillipshmcts) [inline test/accessibility/skip-link.test.ts:20]: Could simplify to use a helper function taking in the template name

**Comment by pr00279** [comment]
> Might be worth extracting this common functionality into a function so not all repeated: const env = nunjucks.configure([ 'node_modules/govuk-frontend', 'views' ]);
> 
> env.addFilter('eval', function(text: string) { return nunjucks.renderString(text, this.ctx); });
> 
> env.addGlobal('i18n', i18n);

I've abstracted some of the repeated elements into a parameterised function! Does this look okay?

**Review by nilay913 — APPROVED** [review]

**Comment by jamiephillipshmcts** [comment]
> > Might be worth extracting this common functionality into a function so not all repeated: const env = nunjucks.configure([ 'node_modules/govuk-frontend', 'views' ]);
> > env.addFilter('eval', function(text: string) { return nunjucks.renderString(text, this.ctx); });
> > env.addGlobal('i18n', i18n);
> 
> I've abstracted some of the repeated elements into a parameterised function! Does this look okay?

Looks good to me!

**Pull request merged**